hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/metrics/TestContainerMetrics.java

```diff
@@ -125,11 +125,11 @@ public void testContainerMetrics() throws Exception {
       client.connect();
 
       // Create container
-      ContainerCommandRequestProto request = ContainerTestHelper
-          .getCreateContainerRequest(containerID, pipeline);
-      ContainerCommandResponseProto response = client.sendCommand(request);
-      Assert.assertEquals(ContainerProtos.Result.SUCCESS,
-          response.getResult());
+//      ContainerCommandRequestProto request = ContainerTestHelper
+//          .getCreateContainerRequest(containerID, pipeline);
+//      ContainerCommandResponseProto response = client.sendCommand(request);
+//      Assert.assertEquals(ContainerProtos.Result.SUCCESS,
+//          response.getResult());
```

Comment:
I feel we should probably remove the step of manually creating container here, since container creation happens only as part of Write_Data phase of writeChunk~?